voice-ui/src/components/App.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect, useState } from 'react';
+import { debugData } from '../utils/debugData';
+import { fetchNui } from '../utils/fetchNui';
+import { useNuiEvent } from '../hooks/useNuiEvent';
+import List from './List';
+
+debugData([
+    {
+        action: "Debug",
+        data: true,
+    },
+]);
+
+interface Voice {
+    voiceModes: Array<[number, string]>;
+    voiceMode: number;
+    radioChannel: number;
+    radioEnabled: boolean;
+    usingRadio: boolean;
+    callInfo: number;
+    talking: boolean;
+}
+
+let EnableUI = ""
+
+const App: React.FC = () => {
+    const [debug, setDebug] = useState(false);
+    const [RadioList, setRadioList] = useState<{ id: string, Name: string, Talking: boolean }[]>([]);
+    const [voice, setVoice] = useState<Voice>({
```

Comment:
With large states like this, useReducer is usually a better choice of readability and maintainability